hack/lib/golang.sh

```diff
@@ -169,8 +167,7 @@ readonly KUBE_STATIC_LIBRARIES=(
   kube-scheduler
   kube-proxy
   kubectl
-  federation-apiserver
-  federation-controller-manager
+  hyperkube
```

Comment:
This is wrong, hyperkube is dynamically linked because of kubelet (until I get my PR working)